assets/script/script.js

```diff
@@ -58,19 +49,20 @@ function fillTagsArray() {
     tagsArray.push(teamData.teams[i].name);
   }
 }
-
+// jQuery UI autocomplete widget
 $(function () {
-  $("#teamSearch").autocomplete({
+  searchInput.autocomplete({
     source: tagsArray,
   });
 });
 
+// render team information
 function renderTeam() {
   for (let i = 0; i < teamData.teams.length; i++) {
```

Comment:
This might be nice as something like this:
```
teamData.teams.filter(it => it.name.toUpperCase() == searchInput.val().toUpperCase() || blah).forEach(it => {
  blah
})
```

It might also be worth pulling out `searchInput.val().toUpperCase()` into a variable: as it is, it's being evaluated twice: jquery stuff is generally quite expensive calls.